environment.yml

```diff
@@ -27,7 +27,8 @@ dependencies:
   - xarray >=2023.11.0
   - xarray-spatial
   - xclim >=0.48.2
-  - xdatasets >=0.3.5
+  - xdatasets >=0.3.6
   - xscen >=0.8.3
   - xvec
   - pip >=23.3.0
+  - pyjuliacall
```

Comment:
I'm guessing here that by adding `pyjuliacall`, we're also pulling `julia` from conda-forge? Is there a base `julia` version we want?